.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+
+
+jobs:
+  build-and-deploy:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    environment:
+      name: production
+      url: ${{ steps.deploy.outputs.url }}
+
+    steps:
+    - name: Checkout code
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)